corehq/apps/reports_core/templates/reports_core/filters/datespan_filter/datespan_filter.js

```diff
@@ -1,6 +1,6 @@
 var filter_id = "#{{ filter.css_id }}-input";
 $(filter_id).createDefaultDateRangePicker();
-$(filter_id).on('apply', function(ev, picker) {
+$(filter_id).on('apply change', function(ev, picker) {
```

Comment:
this brings ucr datepicker usage in line with standard reports. its not totally related to the bug but now it works like everything else